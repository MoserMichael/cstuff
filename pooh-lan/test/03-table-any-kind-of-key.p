# table with any kind of keys

a = { [1,2,3] : 'a' , [2,3,4] : 'b', [3, 4, 5] : 'c', { 'a' : [1, 2, 3], 'b' : 'c' } : 'd' }

println( ~msg a{ [1, 2, 3] } )
println( ~msg a{ [2, 3, 4] } )
println( ~msg a{ [3, 4, 5] } )

println( ~msg a{  { 'a' : [1, 2, 3], 'b' : 'c' } } )
 
