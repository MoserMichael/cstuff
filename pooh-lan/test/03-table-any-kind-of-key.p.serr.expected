003|a = { [ 1 , 2 , 3]  : 'a' , [ 2 , 3 , 4]  : 'b' , [ 3 , 4 , 5]  : 'c' , { 'a' : [ 1 , 2 , 3]  , 'b' : 'c' } : 'd' }
005|println( ~msg a{[ 1 , 2 , 3] }:'a' )...
006|println( ~msg a{[ 2 , 3 , 4] }:'b' )...
007|println( ~msg a{[ 3 , 4 , 5] }:'c' )...
009|println( ~msg a{{ 'a' : [ 1 , 2 , 3]  , 'b' : 'c' }}:'d' )...
